Composer/packages/client/src/pages/botProject/PublishTargets.tsx

```diff
@@ -83,6 +84,26 @@ export const PublishTargets: React.FC<PublishTargetsProps> = (props) => {
     }
   }, [scrollToSectionId]);
 
+  const onDeletePublishTarget = useCallback(
+    async (publishTarget: PublishTarget) => {
+      if (publishTargets) {
```

Comment:
how can it not have value, user just click delete from a list of them

Reply:
Since it is async, I guess if the component was unmounted just as the callback happened it could be undefined.  It should not really be possible, but I get a compile error if I don't check.